### ISSUE TITLE
Specify what is ag__ in the output of tf.autograph.to_code

### ISSUE THREAD

**Issue by ricvo** [tensorflow/tensorflow#53506]
## URL(s) with the issue:
https://www.tensorflow.org/api_docs/python/tf/autograph/to_code

## Description of issue (what needs changing):
I tried to look everywhere but I could not find what is `ag__`, how could I use the output of `tf.autograph.to_code` with the python interpreter.
I thought the name `ag__` was `tf.autograph` but this module does not have `FunctionScope` for example.
tf.compat.v1.autograph also did not work.
I did not seem to be able to find online any explanation of this shorthand `ag__`

### Parameters defined
Could this variable be defined to know how can we interpret the outputted source code?

**Comment by sushreebarsa** [comment]
@ricvo 
In order to expedite the trouble-shooting process here,Could you please fill the issue [template](https://github.com/tensorflow/tensorflow/issues/new/choose),
Thanks!

**Comment by ricvo** [comment]
I am not sure.. you mean the following?

## URL(s) with the issue:
https://www.tensorflow.org/api_docs/python/tf/autograph/to_code

Please provide a link to the documentation entry, for example:
https://www.tensorflow.org/api_docs/python/tf/autograph/to_code

## Description of issue (what needs changing):
I tried to look everywhere but I could not find what is ag__, how could I use the output of tf.autograph.to_code with the python interpreter.
I thought the name ag__ was tf.autograph but this module does not have FunctionScope for example.
tf.compat.v1.autograph also did not work.
I did not seem to be able to find online any explanation of this shorthand ag__

### Clear description
I tried to look everywhere but I could not find what is ag__, how could I use the output of tf.autograph.to_code with the python interpreter.
This would be really useful for anybody wanting to use the source code output of autograph, to be able to have the python interpreter correctly executing it

### Correct links
https://www.tensorflow.org/api_docs/python/tf/autograph/to_code

Example of output of the simple tanh loop in the tutotial for tf.function online:
```
@tf.function(autograph=False)
def tf__tanh_loop(x):
    with ag__.FunctionScope('tanh_loop', 'fscope', ag__.ConversionOptions(recursive=True, user_requested=True, optional_features=(), internal_convert_user_code=True)) as fscope:
        do_return = False
        retval_ = ag__.UndefinedReturnValue()

        def get_state():
            return (x,)

        def set_state(vars_):
            nonlocal x
            (x,) = vars_

        def loop_body():
            nonlocal x
            ag__.converted_call(ag__.ld(tf).print, (ag__.ld(x),), None, fscope)
            x = ag__.converted_call(ag__.ld(tf).tanh, (ag__.ld(x),), None, fscope)

        def loop_test():
            return (ag__.converted_call(ag__.ld(tf).reduce_sum, (ag__.ld(x),), None, fscope) > 1)
        ag__.while_stmt(loop_test, loop_body, get_state, set_state, ('x',), {})
        try:
            do_return = True
            retval_ = ag__.ld(x)
        except:
            do_return = False
            raise
        return fscope.ret(retval_, do_return)
```

### Parameters defined
Parameters are defined.

### Returns defined
The output is somewhat undefined. It should be the source code output of autograph, the problem is that some symbols are not clearly defined and thus the python interpreter doesn't know how to handle them. Example: `ag__`

The following did not work:
`ag__ = tf.autograph`
`ag__ = tf.autograph.operators`
`ag__ = tf.compat.v1.autograph`


### Raises listed and defined
yes

### Usage example
yes


### Request visuals, if applicable
not applicable

### Submit a pull request?
maybe

**Comment by sushreebarsa** [comment]
@ricvo 
This is not a bug or feature request ,could you please refer to the [link1](https://www.tensorflow.org/api_docs/python/tf/autograph/to_code) , [link2](https://www.tensorflow.org/guide/function) and for any further queries you may open this issue in [TF discussion forum ](https://discuss.tensorflow.org/)as there is a larger community there to get you the right help. Thank you!

**Comment by ricvo** [comment]
@sushreebarsa thanks for the time you are dedicating to this issue.
I did not open this as a bug nor as a feature request. I think I opened this as a Documentation Issue. Indeed it is not specified in the Documentation what is `ag__` (see also in the links you suggest).
Working on another issue https://github.com/tensorflow/tensorflow/issues/43800 I found out what `ag__` is, but unfortunately it is not something accessible from outside due to the fact that the module is dynamic it seems.
At the moment I modified my local tensorflow python package code to be able to access the module. The module is accessible internally as
```
tensorflow.python.autograph.impl.api.PyToTF().get_extra_locals()['ag__']
```
as also mentioned in the issue referred above.
I can live with this change, but it's not ideal, I think a more friendly way to access `ag__` should be provided and documented.
I did it locally, seems like a minimal change, I am willing to contribute if needed. Thanks
